File: ios/Objective-C/MesiboObjCSample/MesiboObjCSample/ViewController.m
//
//  ViewController.m
//  MesiboObjCSample
//
//  Copyright © 2020 Mesibo. All rights reserved.
//

#import "ViewController.h"
#import "MesiboSample.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)OnSendMessage:(id)sender {
    [[MesiboSample getInstance] sendMessage:@"destination" message:@"Hello, mesibo"];
}

@end

File: ios/Swift/MesiboSwiftSample/MesiboSwiftSample/ViewController.swift
//
//  ViewController.swift
//  MesiboSwiftSample

//  Copyright © 2020 Mesibo. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func OnSendMessage(_ sender: Any) {
        MesiboSample.getInstance().sendMessage("destination address", message: "Hello, mesibo")
    }
    
}

